frontend/src/toolbar/utils.ts

```diff
@@ -30,17 +30,20 @@ export function getSafeText(el: HTMLElement): string {
     return elText
 }
 
-export function elementToQuery(element: HTMLElement): string | undefined {
+export function elementToQuery(element: HTMLElement, dataAttributes: string[]): string | undefined {
     if (!element) {
         return
     }
+    if (!simmer) {
```

Comment:
Q: Why the lazy init?

Reply:
Cause we don't have the dataAttributes when the JS is evaluated on the root level.